src/tests/fixtures/externals/nonGratsPackageWrapper.ts

```diff
@@ -0,0 +1,19 @@
+// { "EXPERIMENTAL__emitResolverMap": true }
+
+import {
+  SomeType as _SomeType,
+  SomeInterface as _SomeInterface,
+} from "./nonGratsPackage.ignore";
```

Comment:
One thing I'd like to add eventually is the ability for a fixture file to model multiple files. I did something similar for Relay:

Example: https://github.com/facebook/relay/blob/main/compiler/crates/relay-compiler/tests/relay_compiler_integration/fixtures/client_mutation_extension.input
Implemented here: https://github.com/facebook/relay/blob/main/compiler/crates/graphql-test-helpers/src/project_fixture.rs

I think that would help for tests like this which need to encode relationships between different files.

Reply:
Will work in a separate PR, together with test fixes.